### ISSUE TITLE
Finder's cache

### ISSUE THREAD

**Issue by 097115** [ideamonk/Rambola#2]
**ideamonk**, before you close your project, could you please guide me, where is Finder's cache located? I've tried `~/Library/Caches/com.apple.finder` but with no result.

thank you.

**Comment by ideamonk** [comment]
Yes, it used to be ~/Library/Caches/com.apple.finder , also currently on Lion and 10.6.8 it's a 25KB folder. Which means we're not gonna benefit. Months ago when I was using Rambola, this folder used to be ~25 MB, making things pretty fast.

I guess thumbnail cache has been shifted to - /private/var/folders/{short-string}/{long-string}/{shorter-string}/com.apple.QuickLook.thumbnailcache

As mentioned here - http://forums.macrumors.com/showthread.php?t=1012718 , hence no speedups. QuickLook Cache is about 130M, could substantially speedup, but not being sure if {short-string} and {long-string} remain unchanged after reboot.
